src/commands/run.tsx

```
@@ -1180,6 +1317,372 @@ async function runWithTui(
+        parallelState.mergeQueue = [...parallelExecutor.getState().mergeQueue];
+        // Task successfully merged — remove from completedLocally set (it's now fully done)
+        // Create new Set so React detects the change
+        const newSet = new Set(parallelState.completedLocallyTaskIds);
+        newSet.delete(event.taskId);
+        parallelState.completedLocallyTaskIds = newSet;
+        break;
+
+      case 'merge:failed':
+      case 'merge:rolled-back':
+        // Refresh merge queue from executor state
+        parallelState.mergeQueue = [...parallelExecutor.getState().mergeQueue];
```

Biome (2.3.13)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)